akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpChallenge.scala

```diff
@@ -13,7 +13,8 @@ final case class HttpChallenge(scheme: String, realm: String,
                                params: Map[String, String] = Map.empty) extends jm.headers.HttpChallenge with ValueRenderable {
```

Comment:
Yeah I think we missed to add some deprecations when fixing such things :-(
Yes, we should add `@deprecated` wherever possible, and if not directly possible to annotate then add a `// FIXME`